### PR TITLE
SOC-1442 Don't allow token-less GET requests for edits

### PR DESCRIPTION
**Pull request by garthwebb**
Note; this is based off branch SOC-1444 as it has new methods needed for this change set.

https://wikia-inc.atlassian.net/browse/SOC-1442

### CODE REVIEW

**Comment by armonr** [comment]
LGTM

File: extensions/wikia/Wall/js/WallBackendBridge.js [hunk 1]
```diff
@@ -155,7 +155,8 @@
 					isreply: isreply,
 					pagetitle: page.title,
 					pagenamespace: page.namespace,
-					convertToFormat: convertToFormat
+					convertToFormat: convertToFormat,
+					token: window.mw.user.tokens.get('editToken')
 				},
 				callback: this.proxy(function (data) {
```
Comment (kvas-damian) [inline extensions/wikia/Wall/js/WallBackendBridge.js:161]: Does this callback support error message returned when token is not valid?

Comment (garthwebb) [inline extensions/wikia/Wall/js/WallBackendBridge.js:161]: @kvas-damian, this extension JS has no error handling.  As an anonymous user, try clicking 'edit' next to an article comment (or Forum after logging out in another tab) and you'll see these errors just get eaten.

I don't think we care enough about this feature to add this new functionality, but if we do its another ticket.

**Comment by kvas-damian** [comment]
It's not optimal from UI perspective, but we're going to replace Forum and Wall by new Discussion product.

LGTM

**Pull request merged**